src/components/SendAsset/SendAsset.js

```diff
@@ -142,86 +155,37 @@ const SendAsset = ({
   const currentValue = parseFloat(amount || 0);
   const isValidAmount = !!amount && isValidNumber(currentValue.toString()); // method accepts value as string
 
-  const updateTxFee = async (specifiedAmount?: number) => {
-    const value = Number(specifiedAmount || amount || 0);
+  const updateTxFee = () => {
+    const value = Number(amount || 0);
     const isCollectible = get(assetData, 'tokenType') === COLLECTIBLES;
 
     // specified amount is always valid and not necessarily matches input amount
-    if ((!isCollectible && ((!specifiedAmount && !isValidAmount) || value === 0)) || !assetData || !selectedContact) {
-      setGettingFee(false);
-      setTxFeeInfo(null);
-      return null;
-    }
-
-    let data;
-    if (isCollectible) {
-      const provider = getEthereumProvider(getEnv().COLLECTIBLES_NETWORK);
-      const {
-        name,
-        id,
-        contractAddress,
-        tokenType,
-      } = assetData;
-      const collectibleTransaction = {
-        from: activeAccountAddress,
-        to: selectedContact.ethAddress,
-        receiverEnsName: selectedContact.ensName,
-        name,
-        tokenId: id,
-        contractAddress,
-        tokenType,
-      };
-      data = await buildERC721TransactionData(collectibleTransaction, provider);
-    }
-
-    const transaction = { recipient: selectedContact.ethAddress, value, data };
-
-    const formattedAssetData = {
-      token: assetData.token || '',
-      contractAddress: assetData.address || '',
-      decimals: assetData.decimals || 18,
-    };
-
-    const estimated = await smartWalletService
-      .estimateAccountTransaction(transaction, formattedAssetData)
-      .then(res => buildTxFeeInfo(res, useGasToken))
-      .catch(() => null);
-
-    if (!estimated) {
-      Toast.show({
-        message: t('toast.transactionFeeEstimationFailed'),
-        emoji: 'woman-shrugging',
-        supportLink: true,
-      });
-      setGettingFee(false);
-      setTxFeeInfo(null);
-      return null;
+    if ((!isCollectible && (!isValidAmount || value === 0)) || !assetData || !selectedContact) {
+      return;
```

Comment:
Does this result in when the user taps on the related UI element - nothing happens? Is it worth just saying something to the user for this condition?

Reply:
No, this is fine here, it just doesn't update the fee on input change if there are not enough data provided. Hope this makes sense.

Reply:
Ahh got it - thanks!